examples/passport-login/src/observers/wire.observer.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application,
+  CoreBindings,
+  inject,
+  lifeCycleObserver,
+  LifeCycleObserver,
+} from '@loopback/core';
+import {setupBindings} from '../wiring';
+
+/**
+ * This class will be bound to the application as a `LifeCycleObserver` during
+ * `boot`
+ */
+@lifeCycleObserver('wire')
+export class WireObserver implements LifeCycleObserver {
```

Comment:
Any reason setting up the bindings using an observer instead of adding `init()` function in the application class directly.

Reply:
This is to demonstrate the possibility of dividing the responsibility into observers instead of flooding the application class.

Reply:
I see thank you. It's better to mention this best practice in docs as well.